Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -13,6 +13,7 @@
 static const NSInteger DEFAULT_TIMEOUT = 3;
 static const NSInteger RETRY_INTERVAL = 3;
 static const NSInteger MAX_RETRIES = 5;
+static const NSInteger APP_READ_INTERVAL = 520000;
```

Comment:
What does this number represent? It's not an hour, day, or week; is it some other significant value?